src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -1,7 +1,7 @@
 {% use "bootstrap_3_layout.html.twig" %}
 
 {% block form_start -%}
-    {% set attr = attr|merge({class: (attr.class|default('') ~ ' form-horizontal')|trim}) %}
+    {% set row_attr = row_attr|merge({class: (row_attr.class|default('') ~ ' form-horizontal')|trim}) %}
```

Comment:
If we have to make this change here, it seems to me that we need to adapt for custom form types that do not provide the `row_attr` variable (and that we need to fall back to `attr` then).